### ISSUE TITLE
Mobile responsive design?

### ISSUE THREAD

**Issue by michaelnavs** [Miodec/monkeytype#1540]
Recently, I was trying to use Monkeytype on my mobile device and notice there were a few pages that did not have 
a good mobile design. I do not know if this is something you guys plan on implementing but I am willing to contribute and try to fix this issue.

**Screenshots** 
<img width="1440" alt="Screen Shot 2021-06-13 at 1 00 06 PM" src="https://user-images.githubusercontent.com/44414214/121817474-4c503000-cc47-11eb-9479-e211ca4bcb0c.png">
<img width="1440" alt="Screen Shot 2021-06-13 at 12 59 46 PM" src="https://user-images.githubusercontent.com/44414214/121817475-4ce8c680-cc47-11eb-98b0-9505bf6adfd0.png">

**Desktop:** 

- OS: [iOS 14.6] 
- Browser [Firefox] 
- Browser Version [34]

**Comment by Miodec** [comment]
Honestly, i don't think i ever got around to doing a full responsive check on the website simply because mobile is not the target device, and there are other things that need to be done.

**Comment by michaelnavs** [comment]
Understandable. This was the first time I got on the site on a mobile device, so I see where you are coming from.\
\
The only reason I was trying to access it from my phone was because a buddy of mine from school was asking what programs I used to practice touch typing (obviously recommend this one).  I wanted to send him a screenshot of my stats but the mobile view was not good enough for my liking.\
\
I know this is not a priority, but I have some free time this summer and willing to help out.

**Comment by UnrealApex** [comment]
> Honestly, i don't think i ever got around to doing a full responsive check on the website simply because mobile is not the target device, and there are other things that need to be done.

I agree, it doesn't really make sense to do typing tests on mobile anyway.

**Comment by michaelnavs** [comment]
@UnrealApex Agreed but doing typing tests was not the issue. The issue was trying to sign in and check my progress so I can backup my reasoning for recommending this site to a friend.

**Comment by UnrealApex** [comment]
@michaelnavs Didn't consider that use case. In that case I think that should be something that should be worked on.

**Comment by michaelnavs** [comment]
@UnrealApex I already told him lol

**Comment by UnrealApex** [comment]
Also tell him there's smooth caret :)

**Comment by nickthaskater** [comment]
> > Honestly, i don't think i ever got around to doing a full responsive check on the website simply because mobile is not the target device, and there are other things that need to be done.
> 
> I agree, it doesn't really make sense to do typing tests on mobile anyway.

I use Samsung DeX as my primary computing device, which includes my Note 10+ docked with my KDB67, MX Master 3, and 34" ultrawide. "Mobile devices" are not just for holding in your hand on the toilet anymore - they can be and are legitimate computers. Desktop interfaces like Samsung DeX solidify this.

Sadly, even when accessing Monkeytype via DeX using either Chrome and Samsung Internet, the old issue of key input not being recognized rears its head, despite using an external physical keyboard.

**Comment by SeerLite** [comment]
@nickthaskater Does it work correctly in the preview page of #1325? You can test it [here](https://seerlite.github.io/monkeytype/issue/531-input-rewrite/)

**Comment by nickthaskater** [comment]
> @nickthaskater Does it work correctly in the preview page of #1325? You can test it [here](https://seerlite.github.io/monkeytype/issue/531-input-rewrite/)

Yes! It works perfectly in Samsung DeX with a physical keyboard. Amazing.
